stories/00-example.md

```diff
@@ -1,10 +1,9 @@
-# Example Title
```

Comment:
pls dont repolace the example md.  pls add a new one file and revert example.md.

Reply:
Fixed it and added a new file.